apps/mymedicare_cb/models.py

```diff
@@ -107,8 +115,8 @@ def __get_and_update_user(mbi, user_id, hicn_hash, request, auth_type, slsx_clie
         'subject': user_id,
         'fhir_id_v2': bfd_fhir_id_v2,
         'fhir_id_v3': bfd_fhir_id_v3,
-        'hicn_hash': hicn_hash,
-        'hash_lookup_type': hash_lookup_type,
+        'hicn_hash': slsx_client.hicn_hash,
```

Comment:
```suggestion
        'hicn_hash': hicn_hash,
```

Same thing here as mbi above to preserve 4294 changes